Traceback (most recent call last):
  File "test/verify/check-services", line *, in testCreateTimer
    m.execute("timedatectl set-ntp off; timedatectl set-time '2020-01-01 15:30:00'")
*
CalledProcessError: Command 'timedatectl set-ntp off; timedatectl set-time '2020-01-01 15:30:00'' returned non-zero exit status 1
